src/bridge.cpp

```diff
@@ -194,11 +195,11 @@ int bridgeAngle(
         }
     }
 
-    if (idx1 < 0 || idx2 < 0)
-        return -1;
+    if (! idx1 || ! idx2)
+        return -1.0;
 
-    Point center1 = islands[idx1].centerOfMass();
-    Point center2 = islands[idx2].centerOfMass();
+    Point2LL center1 = islands[*idx1].centerOfMass();
+    Point2LL center2 = islands[*idx2].centerOfMass();
```

Comment:
```suggestion
    Point2LL center1 = islands[idx1.value()].centerOfMass();
    Point2LL center2 = islands[idx2.value()].centerOfMass();
```

Reply:
You don't like the `*` syntax of `std::optional` ? :cry: